gateway/gateway_test.go

```diff
@@ -579,9 +527,8 @@ func TestClose(t *testing.T) {
 		_, err = g.logReader.Read(buf)
 		test.That(t, err, test.ShouldNotBeNil)
 	}
-	if g.dataFile != nil {
-		buf := make([]byte, 1)
-		_, err = g.dataFile.Read(buf)
+	if g.db != nil {
```

Comment:
isn't this impossible?

Reply:
yeah, tbh was just using the same setup that was here, but none of these should be `nil` in this test